Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/SailPointIdentityNow.py

```diff
@@ -0,0 +1,622 @@
+
+    :type request_type: ``str``
+    :param request_type: Type of access request(GRANT_ACCESS or REVOKE_ACCESS).
+
+    :type requested_for: ``str``
+    :param requested_for: Identity Id for whom the access request is being made.
+
+    :type requested_item: ``str``
+    :param requested_item: Id of the object(access profile or role).
+
+    :type requested_item_type: ``str``
+    :param requested_item_type: Type of object(ACCESS_PROFILE or ROLE).
+
+    :type comment: ``str``
+    :param comment: Comments to attach to the item request.
+
+    :return: Success or failure status.
+    """
+    if request_type is None or requested_for is None or requested_item is None or requested_item_type is None:
+        return None
+
+    json_data = {
+        "requestedFor": [requested_for],
+        "requestType": request_type,
+        "requestedItems": [
+            {
+                "type": requested_item_type,
+                "id": requested_item,
+                "comment": comment
+            }
+        ],
+        "clientMetadata": {}
+    }
+    response = client.send_request(IDN_BETA_ACCESS_REQUEST_EXT, "POST", None, json_data)
+    if response is not None and 200 <= response.status_code < 300:
+        return 'Access request was successful!'
+    elif 'detailCode' in response.json():
+        return response.json()
+    return None
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    """
+    Intercept and execute commands.
+    """
+
+    # IdentityNow API Base URL (https://org.api.identitynow.com)
+    base_url = demisto.params().get('identitynow_url')
+
+    # OAuth 2.0 Credentials
+    client_id = demisto.params().get('client_id')
+    client_secret = demisto.params().get('client_secret')
+    grant_type = 'client_credentials'
+
+    # Convert the argument to an int or set to MAX_INCIDENTS_TO_FETCH
+    max_results = int(demisto.params().get('max_fetch'))
+    if not max_results or max_results > MAX_INCIDENTS_TO_FETCH:
+        max_results = MAX_INCIDENTS_TO_FETCH
+
+    # first_fetch_str = demisto.params().get('first_fetch', '3 days')
+
+    # Other configs
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    request_timeout = 10
+
+    headers = get_headers(base_url, client_id, client_secret, grant_type)
+    client = Client(
+        base_url=base_url,
+        verify=verify_certificate,
+        proxy=proxy,
+        headers=headers,
+        max_results=max_results,
+        request_timeout=request_timeout)
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        results = None
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            results = test_connection(base_url, client_id, client_secret, grant_type)
+
+        elif demisto.command() == 'identitynow-search-identities':
+            query = demisto.args().get('query', None)
```

Comment:
```suggestion
            query = demisto.args().get('query')
```

None is already the default value of 'get', you don't have to supply it here